packages/logs/lib/models/helpers.ts

```diff
@@ -50,8 +42,45 @@ export function getFormattedMessage(
         syncConfigName: syncConfig?.name || data.syncConfigName || null,
 
         jobId: data.jobId || null,
+        meta: meta || data.meta || null,
 
         userId: user?.id || data.userId || null,
+        parentId: null
+    };
+}
+export function getFormattedMessage(data: Partial<MessageRow>, { meta }: FormatMessageData = {}): MessageRow {
```

Comment:
should this method return a `Pick<MessageRow, ...>` to make it clear it is only setting a subset of the fields?

Reply:
it's still returning everything, just more null than before :D

Reply:
forget what I said. I thought the function was only used in getFormattedOperation but it is called from a lot of places